integrationTests/multiShard/smartContract/polynetworkbridge/bridge_test.go

```diff
@@ -102,18 +105,65 @@ func TestBridge_Setup(t *testing.T) {
 		txData,
 		integrationTests.AdditionalGasLimit,
 	)
-	_, _ = integrationTests.WaitOperationToBeDone(t, nodes, 6, nonce, round, idxProposers)
+	nonce, round = integrationTests.WaitOperationToBeDone(t, nodes, 6, nonce, round, idxProposers)
 
 	scQuery := &process.SCQuery{
 		CallerAddr: ownerNode.OwnAccount.Address,
 		ScAddress:  scAddressBytes,
 		FuncName:   "getWrappedEgldTokenIdentifier",
 		Arguments:  [][]byte{},
 	}
-	vmOutput, _ := ownerNode.SCQueryService.ExecuteQuery(scQuery)
+	vmOutput, err := ownerNode.SCQueryService.ExecuteQuery(scQuery)
+	require.Nil(t, err)
 	require.NotNil(t, vmOutput)
-	fmt.Println()
-	fmt.Println(vmOutput.ReturnData)
 	require.NotZero(t, len(vmOutput.ReturnData[0]))
-	require.Equal(t, []byte("WEGLD"), vmOutput.ReturnData[0][:4])
+
+	tokenIdentifier := vmOutput.ReturnData[0]
+	require.Equal(t, []byte("WEGLD"), tokenIdentifier[:5])
+
+	burntValue := big.NewInt(5)
```

Comment:
`valueToBurn` ? It is not burnt yet when declaring the variable

Reply:
changed